src/weekly-notes/action.yml

```
@@ -2,14 +2,16 @@ name: 'Open weekly issue'
 description: 'Create new release issue when the last one was closed'
 inputs:
   token:
-    description: 'GITHUB_TOKEN or a repo scoped PAT.'
+    description: 'GITHUB_TOKEN or a repo scoped PAT'
     default: ${{ github.token }}
   template-path:
     description: 'Relative path to the weekly template'
     required: true
     default: '.github/ISSUE_TEMPLATE/WEEKLY_NOTE.md'
   moderators-path:
     description: 'Relative path to the list of moderators'
+  community-worker:
+    description: 'Community worker = true|false. Set to 'false' to not assign a community worker, will default to true'
```

The current behavior around input parameters in GHA is a bit opaque and relies on not explicitly documented behavior of `getInput` returning an empty string in such case (https://github.com/orgs/community/discussions/26777), so I'd suggest going for an explicit approach, to have a more robust behavior when skipping `required,`  and `default` are not specified:
```suggestion
    description: 'Community worker = true|false. Set to 'false' to not assign a community worker, will default to true'
    type: boolean
    required: false
    default: true
```

This is to ensure all the end users of this action omitting the `community-worker` input parameter are guaranteed the same result in the future.

---

@MaxTru Some additional helpful feedback. Maybe you have time to follow up?

---

@maxdanilov All good, happy to ship twice.

---

Incorporated via https://github.com/bpmn-io/actions/pull/7.